### DISCUSSION TITLE
Incompatible Layer Question

### DISCUSSION THREAD

**Discussion by AlphaBears** [nftchef/art-engine#263] [Help + Q&A]
Hello! Thanks so much for this repo, I have a question that I'm currently getting stuck on.

I have laser eyes that can't be in the eye folder, because the eye folder comes before the headwear folder and the laser eyes clip with hats and hair. How would I go about making that an exception?

**Comment by BarryDB83** [comment]
1) put eyes above headwear 
2) prevent certain hair/laser or hat/laser combinations with the incompatible function (well documented in readme)
3) if only custom designs have lasers then it's a non-issue as you will design these manually and they get mixed into the lot afterwards
